000-vip-init.php

```diff
@@ -180,8 +180,7 @@
 	$headers['X-Powered-By'] = 'WordPress.com VIP <https://wpvip.com>';
 
 	// All non-production domains should not be indexed.
-	// This should not apply only to *.vip-go.co
-	if ( 'production' !== VIP_GO_ENV ) {
+	if ( 'production' !== VIP_GO_ENV || false !== stristr( $_SERVER[ 'HTTP_HOST' ], '.go-vip.' ) ) {
```

Comment:
`strpos` is generally preferred over `strstr`, especially when we only care about checking if the string has something:

```suggestion
	if ( 'production' !== VIP_GO_ENV || false !== strpos( $_SERVER[ 'HTTP_HOST' ], '.go-vip.' ) ) {
```